src/Peering/Peering/help/Set-AzPeeringDirectConnectionObject.md

```diff
@@ -177,6 +196,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -UseForPeeringService
+Enable for use with Microsoft Peering Service (MPS).
+
+```yaml
+Type: System.Nullable`1[System.Boolean]
```

Comment:
@rileymckenna is this cmdlet following `PUT` or `PATCH` semantics? If it's `PUT`, then this should be a switch parameter.

Reply:
this is an in memory command, so it doesn't actually do a put. Just creates direct connection object.